.gitignore

```diff
@@ -34,4 +34,6 @@ ehthumbs_vista.db
 [Dd]esktop.ini
 
 # Development files
-node_modules/
\ No newline at end of file
+node_modules/
+addons/find-bar/userscript.js
+addons/find-references/userstyle_1.css
```

Comment:
```suggestion
```
Please don't modify `.gitignore`, the old files are still in git's history. If you want to keep them around locally just don't commit those files or move them to the desktop.